include/srtp.h

```diff
@@ -1855,6 +1855,29 @@ srtp_err_status_t srtp_get_protect_rtcp_trailer_length(srtp_t session, uint32_t
 
 
 /**
+ * @brief srtp_set_stream_roc(session, ssrc, roc)
```

Comment:
Please document the parameters.  I'd personally prefer the ssrc to be provided in host byte order.  (We have one function that assumes network order and I think that's really unfortunate.)